### PR TITLE
Attempt to Add a second civilian shuttle on atlas

### PR DESCRIPTION
**Pull request by DeltaDav**
## About The Pull Request

Okay, lil warning, I might need a bit of help in order for it to work, since im not quite familliar with this code, shuttle code, Or heck, even coding in a general. Only Mapping.

I was also unable to test it on local server, Dream Daemon got a issue. 

This adds a second Shuttle meant for civilian use. The Centuries Old Shuttle. Basically, its a cheap replica of a old earth shuttle, retrofitted for mid-range space travel. Its consumes more fuel that the civilian shuttle, but is bigger, has a cargo hold, sleeper (survival pod one), toilet and shower, jukebox and coffee vendromat and comfier "lounge" setting... Smells bad and old tho. 

For transparency, here is the extra details added :
- the light bulbs are all ORANGE (because older)
- one vodka bottle in the toilet / showers
- 2 toys on the consoles cockpit
- Has a Fax machine
- Has a sector holopad
- Has a Pilot Laptop
- Has space teddi bear on the seats, with a iced coffee can 
- Cargo hold has a few crates with > Russian outfit and hat > Hawaii shirts > Russian Emergency voidsuit > Improvised jetpack

2 phoron cannister can be connected. Rest is a copy paste from the Civilian Shuttle

Its on the West Northern Pad, north of the spec ops pad, west the trader pad. Electric line Added, on the Shuttle SMES network. 

Major flaws : A side door that can be activated by anyone, even in mid space, no scrubber, no generator, and fuel hungry (compared to the civilian one.)

## Why It's Good For The Game

I understood, maybe wrongly, but that it planned to give more things to do outside the station. A second shuttle can be a way to welcome those comming change and direction.
Also gives more freedom to pilot who likes to change the layout of the ship, since its bigger. (Okay I only saw 1 guy appart of me doing that) 

image

## Changelog

:cl:
add: Shuttle to map
add: Shuttle to code
del: Nope
tweak: Nope
qol: Nope
balance: Nope
fix: Nope
soundadd: Nope
sounddel: Nope
imageadd: Nope
imagedel: Nope
spellcheck: Nope, but there might be some (ESL)
code: Added things to code, basically Copy Pasted
refactor: Nope
config: Nope
admin: Nope
server: I am unable to test it.
/:cl:

### CODE REVIEW

**Comment by TheLordME** [comment]
You are supposed to delete the Changelog entries that dont apply to your PR.
for this a simple
add: Added a second civilian shuttle to the Triumph map
is probably enough

**Comment by TheLordME** [comment]
please dont use diagonal windows:
grafik
1. they dont connect 
2. they are not garunteed to be/stay airtight

**Comment by TheLordME** [comment]
afaik, we moved away from the shuttle walls as walls(since they arent baywall compatible)
grafik

**Comment by DeltaDav** [comment]
Understood, will change the windows layout and walls.
Will also remove some engines, this old shuttle might be a bit too fast with the 8.
Are there any issue I need to fix code wise? Or things I must be careful with?

**Comment by TheLordME** [comment]
Haven't looked at the code yet

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -294,6 +294,17 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 /area/shuttle/civvie/cockpit
 	name = "\improper Civilian Transport Cockpit"
 
+///Centuries old Civilian Shuttle
```
Comment (TheLordME) [inline code/game/area/Space Station 13 areas.dm:297]: I dislike the naming, perhaps something like "Century shuttle" would be better (century old suggests that its very outdated)

**Review by TheLordME — COMMENTED** [review]
I think the name needs a rethink, otherwise, I am seeing no immidet issues

File: code/game/area/Space Station 13 areas.dm (hunk 1 above)
Comment (DeltaDav) [reply code/game/area/Space Station 13 areas.dm]: Changed the name,

image

**Comment by TheLordME** [comment]
I understood, that the intention is to make it sound old, but centuries would be ancient (the time difference between now and ingame now is 500 something years) which could be refered to as centuries old.

**Comment by silicons** [comment]
looking at this soon

**Comment by DeltaDav** [comment]
So after test merge, here are a few things that I might need help fixing (ill attempt on my own too) :
- All consoles doesn't connect to the ship parts, rendering it completly unusable
- The Airlock seems to be a bit buggy while cycling out
- Shuttle invisible to other on sensor 

Mapping wise, and issues I am fixing right now : 
-Wrong type of curtain (no sprite after use)  
- Air lock Buttons missplaced (but useable)
- Can't escape the sleeper :D (scream internaly)

Other issues:
- Shuttle is too big for the nebula gas station (but since there is a jetpack, emergency voidsuit and the nebula external airlocks works, a pilot can EVA out.)

might have missed some

**Comment by DeltaDav** [comment]
okay the last 3 commit were to fix a few issues.
They also had chairs and hand rails to the cargo bay.
And a transit position, completly forgot about it

**Comment by DeltaDav** [comment]
should be fixed

**Comment by BlueWildrose** [comment]
The window tiles have snow pouring in them, suggesting there is no roofing placed there. Add some roofing there.
dreamseeker_2024-04-17_22-35-01

**Review by TheLordME — CHANGES_REQUESTED** [review]
Please don't add extra areas to space station 13 areas.dm
Add a new file to code/game/area/station
Something like shuttle.dm

**Review by TheLordME — APPROVED** [review]

**Pull request merged**